_source/_includes/toc.md

```diff
@@ -0,0 +1,7 @@
+{%- if page.path contains '.adoc' -%}
+:toc: macro
+{%- else -%}
+**Table of Contents**{: .hide }
+* Table of Contents
+{:toc}
+{%- endif -%}
```

Comment:
On 2nd thought, maybe we shouldn't add support for AsciiDoc here since its TOC macro is shorter than the include. 😅

Reply:
ha ha, but for sake of consistency I think its ok

Reply:
I added this specifically to avoid markdown format messing it up as it turns the `*` in the template to `-` and breaks it. I was caught by it twice so :wink: